internal/runner/runner.go

```diff
@@ -43,7 +47,7 @@ type (
 
 func NewRunner(natsClient NatsClient, hops *dsl.HopsFiles, logger zerolog.Logger) (*Runner, error) {
 	runner := &Runner{
-		hopsKey:     fmt.Sprintf("hopsconf-%s", hops.Hash),
+		hopsKey:     hopsKeyFromHash(hops.Hash),
```

Comment:
This feels like indirection for the sake of it. The only place we calculate this key is here, and that fact is intrinsic in how the key is used (it's a key for the local hops held by the runner).

Reply:
In fact, looking at the below where we're transforming between the key and the hash repeatedly... I'm not sure I see the point at all.

We could just use the hash. I think I added they prefix so if I read that `hops` message I'd understand what I was looking at, but on reflection it doesn't help with that and even if it did it's a pointless goal.

Let's cut my nonsense out entirely and just use the hash.

Reply:
This will break all stored contexts, meaning any long running task won't work anymore. Not a current problem however. So won't put in a fallback.

Reply:
Changed. Will just use hash.